modules/core/src/lib/deck.js

```diff
@@ -158,6 +156,10 @@ export default class Deck {
     }
     this.animationLoop = this._createAnimationLoop(props);
 
+    // TODO(Tarek): Eventually, stats object should come from Luma.gl
+    //   Frame rate stat in _onRenderFrame can be removed when that happens.
+    this.stats = new Stats({id: 'deck.gl'});
```

Comment:
Here `deck.gl: ` could be added to any string we present to make things clear...